### PR TITLE
Updated Policy Documentation for Multiple Policy Assessment.

### PR DESCRIPTION
**Pull request by jaiswalricha**
Updated Policy Documentation for Multiple Policy Assessment.

### CODE REVIEW

File: Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md [hunk 1]
```diff
@@ -134,6 +134,17 @@ Following policy will configure/Audit security contacts (email) at Subscription
 
 ___ 
 
+#### Policy Assessment Evaluation
+
+To collect the complete audit evaluation for a control, please run the policies in below order.
+
+The assessment result should be combination of:
```
Comment (ZiaAhmad-PM) [inline Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md:141]: The assessment result should be a combination of:

File: Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md [hunk 2]
```diff
@@ -134,6 +134,17 @@ Following policy will configure/Audit security contacts (email) at Subscription
...
+
+1. [Security - SQL Server - AuditSqlServerAuditingSettings](Security%20-%20SQL%20Server%20-%20AuditSqlServerAuditingSettings.json)
+2. [Security - SQL Server - AuditSqlServerSecuritySettings](Security%20-%20SQL%20Server%20-%20AuditSqlServerSecuritySettings.json)
+3. [Security - SQL Server - AuditSqlServerThreatDetection](Security%20-%20SQL%20Server%20-%20AuditSqlServerThreatDetection.json) OR [Security - Subscription - UpdateSecurityContacts](Security%20-%20Subscription%20-%20UpdateSecurityContacts.json)
```
Comment (ZiaAhmad-PM) [inline Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md:145]: Is it [(1) and (2) and (3a)] OR [(1) and (2) and (3b)]?

File: Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md [hunk 3]
```diff
@@ -134,6 +134,17 @@ Following policy will configure/Audit security contacts (email) at Subscription
...
+___ 
+
 
 ### Notes
 1. It is recommended to assign policy to setup MDC security contacts at Subscription scope (or Management group with subscriptions managed by same team) as it will configure same email address for all subscriptions in scope.
```
Comment (ZiaAhmad-PM) [inline Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md:150]: 1. It is recommended to assign the policy to audit the MDC security contacts configuration at Subscription scope (or Management group with subscriptions managed by the same team) as the policy will configure the same email address for all subscriptions in a given MG scope.

I hope that is what you meant by highlighting the "scope" dimension?

**Review by ZiaAhmad-PM — CHANGES_REQUESTED** [review]

File: Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md (hunk 2 above)
Comment (jaiswalricha) [reply Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md:145]: Hi Zia, the result should be combination of 1 +2 + 3A/ 1+2+3B

Comment (jaiswalricha) [reply Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md:145]: that's why I placed the 'OR' in point num 3

File: Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md (hunk 3 above)
Comment (jaiswalricha) [reply Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md:150]: Yes Zia, highlighted the Audit thing

File: Policies/APIManagement/Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN/Readme.md [hunk 4]
```diff
@@ -36,8 +36,21 @@ Param Name|Description|Default Value|Mandatory?
...
 | Effect | The effect determines what happens when the policy rule is evaluated to match| Audit |No |
 
+
+#### Policy Assessment Evaluation
+
+To collect the complete audit evaluation for a control, please run the policies in below order.
+
+The assessment result should be a combination of:
+
+1. [Security - APIManagement - Deny API Management Services with Basic Authentication Enabled](Security%20-%20APIManagement%20-%20Deny%20API%20Management%20Services%20with%20Basic%20Authentication%20Enabled.json)
```
Comment (TarunKrShukla) [inline Policies/APIManagement/Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN/Readme.md:46]: these two should be different ryt?

**Review by TarunKrShukla — APPROVED** [review]

**Review by ZiaAhmad-PM — APPROVED** [review]
Approved

**Pull request merged**